Frameworks/CoreMotion/CMMotionManager.mm

```diff
@@ -15,34 +16,124 @@
 #import <StubReturn.h>
+#import "CMAccelerometerDataInternal.h"
+#import "CMMotionManagerWinRT.h"
+
+typedef wchar_t WCHAR;
+#import "UWP/WindowsDevicesSensors.h"
 
 NSString* const CMErrorDomain = @"CMErrorDomain";
 
-@implementation CMMotionManager
+
+@interface CMMotionManager () 
+@property WDSAccelerometer* accelerometer;
+@property EventRegistrationToken accelerometerToken;
+@property NSOperationQueue* accelerometerQueue; 
+@property (readwrite, nonatomic, getter=isAccelerometerActive) BOOL accelerometerActive;
+@property (readwrite, nonatomic, getter=isAccelerometerAvailable) BOOL accelerometerAvailable;
+@property (readwrite) CMAccelerometerData* accelerometerData; 
+@end
+
+
+@implementation CMMotionManager 
+
+- (instancetype)init {
+    if (self = [super init]) {        
+        
+        // Calling into C++/CX to determine if the default accelerometer device is available
+        // This step is necessary for now as this call using projections [WDSAccelerometer getDefault] does not return NULL
+        // as expected for the case where no accelerometer device is available
+       if (!checkDefaultAccelerometer()) {                
+            _accelerometerAvailable = false;
+            _accelerometerActive = false;
+            _accelerometerUpdateInterval = 0;
+        } else {
+            _accelerometer = [WDSAccelerometer getDefault];
+            _accelerometerAvailable = true;
+            _accelerometerActive = false;
+            _accelerometerUpdateInterval = _accelerometer.minimumReportInterval;
+            _accelerometer.reportInterval = _accelerometer.minimumReportInterval;
+        }
+    } 
+
+    return self;
+}
+
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
-- (void)startAccelerometerUpdatesToQueue:(NSOperationQueue*)queue withHandler:(CMAccelerometerHandler)handler {
-    UNIMPLEMENTED();
+- (void)startAccelerometerUpdatesToQueue:(NSOperationQueue*)queue 
+                                           withHandler:(CMAccelerometerHandler)handler {
+    if (self.accelerometerAvailable) {
+        [self startAccelerometerUpdates];
+        self.accelerometerQueue = queue;
+
+        self.accelerometerToken = [self.accelerometer addReadingChangedEvent:
+            ^void(WDSAccelerometer* sender, WDSAccelerometerReadingChangedEventArgs* e) {
+
+                CMAccelerometerData* data = [[CMAccelerometerData alloc] initWithValues:e.reading.accelerationX 
+                                                                                      y:e.reading.accelerationY
+                                                                                      z:e.reading.accelerationZ];
+                [queue addOperationWithBlock:^{ handler(data, nil); }];                                   
+            }];          
+    }    
 }
 
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 - (void)startAccelerometerUpdates {
-    UNIMPLEMENTED();
+    
+    if (self.accelerometerAvailable) {
```

Comment:
Any threading concerns on any of these accessors/methods?  Should we synchronize on self?